util/llbutil/authprovider/cloudauth/ecr_login.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/docker/cli/cli/config/types"
 )
 
+const (
+	ECRCredHelper = "ecr-login"
```

Comment:
Using a constant but I can undo this change (made it to match with `GCPCredHelper`)

Reply:
nice change!